iOS/SampleApp/AppDelegate.m

```diff
@@ -5,21 +5,20 @@
 //
 
 #import "AppDelegate.h"
-#import <OTAcceleratorPackUtil/OTAcceleratorPackUtil.h>
+#import <OTAcceleratorPackUtil/OTAcceleratorSession.h>
```

Comment:
You don't actually need `<` and `>`. it's completed opened. using double quotes should be fine.